src/javascript/app/pages/cashier/dp2p.js

```diff
@@ -0,0 +1,95 @@
+const React        = require('react');
+const ReactDOM     = require('react-dom');
+const Client       = require('../../base/client');
+const BinarySocket = require('../../base/socket');
+const getLanguage  = require('../../../_common/language').get;
+const urlForStatic = require('../../../_common/url').urlForStatic;
+
+const DP2P = (() => {
+
+    const onLoad = () => {
+        const is_svg = Client.get('landing_company_shortcode') === 'svg';
+        if (is_svg) {
+            import('@deriv/p2p').then((module) => {
+                const el_dp2p_container = document.getElementById('binary-dp2p');
+                const shadowed_el_dp2p = el_dp2p_container.attachShadow({ mode: 'closed' });
+
+                const el_main_css = document.createElement('style');
+                el_main_css.innerHTML = `
+                @import url(${urlForStatic('css/p2p.min.css')});
+                :host {
```

Comment:
can we put it inside scss instead?

Reply:
@mustofa-binary We cannot 👍 The styles should be placed inside the shadow DOM which we access through JS, and we need to get the path of the CSS file with `urlForStatic` function.

Reply:
can you make a comment on it then, since its an unusual flow on BS

Reply:
Sure I'll add it @mustofa-binary